phing/build.yml

```diff
@@ -72,11 +72,18 @@ git:
     pre-commit: ${blt.root}/scripts/git-hooks
     commit-msg: ${blt.root}/scripts/git-hooks
 
+# Site(s) definition.
 multisite:
-  # The docroot/sites/default directory is used by default.
-  name:
-    - default
-    # - another_site
+  default:
+    aliases:
+      local: self
+      remote: ${project.machine_name}.test
+    uri:
+      local: ${project.machine_name}.local
```

Comment:
The reason I have both here right now is for the `local:sync` target. I'm not aware of a way to do a `sql-sync` only specifying `uri`s, but perhaps there's a way I don't know of?